experimental/PropertyDDS/packages/property-common/src/events.ts

```diff
@@ -1,543 +0,0 @@
-/*!
```

Comment:
Why has this file been removed? It is not needed anywhere?

Reply:
Using built-in "events" package from TS.